ec/Cargo.toml

```diff
@@ -19,8 +19,13 @@ ark-ff = { path = "../ff", default-features = false }
 derivative = { version = "2", features = ["use_core"] }
 num-traits = { version = "0.2", default-features = false }
 rayon = { version = "1", optional = true }
+serde_json = { version = "1.0.58", optional = true }
+paste = "0.1"
 zeroize = { version = "1", default-features = false, features = ["zeroize_derive"] }
 
+[dev-dependencies]
+rand_xorshift = "0.2"
```

Comment:
We can use `ark_std::rand::XorShiftRng` directly, no need to import this.
```suggestion
```

Reply:
yeah I'll have to change this...